EMF/common/scripted_triggers/emf_core_triggers.txt

```diff
@@ -180,6 +180,26 @@ emf_divine_blood_religion = {
 	}
 }
 
+emf_same_basic_religion_as_ROOT = {
+	OR = {
+		religion = ROOT
+		is_heresy_of = ROOT
+		is_parent_religion = ROOT
+		is_reformed_religion = ROOT
+		ROOT = { is_reformed_religion = PREV }
+	}
+}
+
+emf_same_basic_religion_as_PREV = {
+	OR = {
+		religion = PREV
+		is_heresy_of = PREV
+		is_parent_religion = PREV
+		is_reformed_religion = PREV
+		PREV = { is_reformed_religion = PREV }
```

Comment:
Are you sure this works? Wouldn't this end up checking `PREVPREV` instead of `THIS`?

Reply:
Yeah, I'm definitely sure.  I put together a whole series of test events to be sure that my model of how PREV-chaining works was indeed still correct (as it's extremely error-prone and confusing). The `PREV` inside of the `PREV = {` simply refers to the previous scope we were in before we switched scopes with `PREV = {`, which is the base scope of the scripted trigger.